apps/demo-game/src/services/ActionsReducer.ts

```diff
@@ -42,6 +44,13 @@ export function apply(state: State, action: Actions) {
       throw new Error('Bank + Bonds + Stocks must equal 100')
 
     draft.result.decisions = action.payload.playerArgs
+
+    // This is only to test the game facts
```

Comment:
would rather add something real to test the game facts and not myInt in one place and actionCounter in the other (inconsistent)

in startinvest, we could e.g. add a "gameDescription" that is shown on first login (random idea)